style.css

```diff
@@ -0,0 +1,88 @@
+:root {
+    --white: white;
+    --black: black;
+    --gray: #d8d9db;
+    --gray-hoover: #ebebeb;
+    --gray-active: #bbbcbe;
+    --orange: #df974c;
+    --orange-hoover: #dfb07e;
+    --orange-active: #dd8d37;
+}
+
+/* Generics */
+
+body {
+    margin: 0;
+    padding: 0;
+    font-family: "Courier New", Courier, monospace;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+/* Calculator */
+
+.calc {
+    width: 400px;
+    background-color: var(--black);
+    color: var(--white);
+}
+
+.screen {
+    color: var(--white);
+    font-size: 40px;
+    text-align: right;
+    padding: 20px 5px;
+}
+
+.calc-row {
+    display: grid;
+    row-gap: 3px;
+    column-gap: 3px;
+    margin-bottom: 0.1em;
+    grid-template-columns: repeat(4, 1fr);
+}
+
+.span-2 {
+    grid-column: span 2;
+}
+
+.span-3 {
+    grid-column: span 3;
+}
+
+.calc-button {
+    background-color: var(--gray);
+    color: var(--black);
+    height: 100px;
+    border: none;
+    border-radius: 0;
+    font-size: 40px;
+    cursor: pointer;
+}
+
+.calc-button:hover {
+    background-color: var(--gray-hoover);
+}
+
+.calc-button:active {
+    background-color: var(--gray-active);
+}
+
+.calc-button:last-child {
```

Comment:
We should not style based on this. Instead we should have a "btn-orange" class that you apply from CSS to the buttons you want to be orange.

So you should have 3 classes: "btn" which is with the default styles, that are common across all buttons.

Then 2 more: "btn-black" and "btn-orange"